images/runtime/php/7.2/7.2.Dockerfile

```diff
@@ -87,7 +87,7 @@ RUN { \
 	&& a2enconf docker-php
 
 ENV PHP_EXTRA_BUILD_DEPS apache2-dev
-ENV PHP_EXTRA_CONFIGURE_ARGS --with-apxs2 --disable-cgi
+ENV PHP_EXTRA_CONFIGURE_ARGS --with-apxs2 --disable-cgi ac_cv_func_mmap=no
```

Comment:
Is the issue happening for all php versions or only some of them? I triggered the validation build and nightly build for the PR, as long as it's not breaking any existing builds and minimize the impact, and make sure we passed all the tests. Thanks!

Reply:
The issue happens with all versions of PHP "blessed" images in VMSS (7.4, 8.0, 8.1, also tested for 7.0) under the Ubuntu Kernel 5.4 with CIFS mountpoints. This does not happen under Kernel 4.15.

**php-src test results:**

With ac_cv_func_mmap=no
```
=====================================================================
TEST RESULT SUMMARY
---------------------------------------------------------------------
Exts skipped    :   42
Exts tested     :   33
---------------------------------------------------------------------

Number of tests : 16476             11607
Tests skipped   : 4869 ( 29.6%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :   45 (  0.3%) (  0.4%)
Expected fail   :   35 (  0.2%) (  0.3%)
Tests passed    : 11527 ( 70.0%) ( 99.3%)
---------------------------------------------------------------------
Time taken      :  640 seconds
=====================================================================
```


Without ac_cv_func_mmap=no
```
=====================================================================
TEST RESULT SUMMARY
---------------------------------------------------------------------
Exts skipped    :   42
Exts tested     :   33
---------------------------------------------------------------------

Number of tests : 16476             11607
Tests skipped   : 4869 ( 29.6%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :   45 (  0.3%) (  0.4%)
Expected fail   :   35 (  0.2%) (  0.3%)
Tests passed    : 11527 ( 70.0%) ( 99.3%)
---------------------------------------------------------------------
Time taken      :  645 seconds
=====================================================================
```